### DISCUSSION TITLE
Unable to send frax from eth metamask to solana sollet

### DISCUSSION THREAD

**Discussion by aldo881** [wormhole-foundation/wormhole#317] [Problems / bugs] [answered]
Ethereum Mainnet -> Solana Mainnet

trying to send Frax ETH on metamask to Frax SOL sollet

I always get this error at the lock assets stage:

Lock assets
🪓 Waiting for MetaMask transfer approval
❌ Transfer failed!

 I tried different metamask accounts and adresses one with a ledger, one without, and the result is the same

when I triy it again it says:

Approve assets
✅ Already approved on ETH!
Lock assets
🪓 Waiting for MetaMask transfer approval
❌ Transfer failed!

any idea what I am doing wrong or can do to make it work?

**Comment by leoluk** [comment, marked as answer]
This looks like a UI issue - please contact the wormholebridge.com developers: https://github.com/solana-labs/oyster/issues
